src/migrations/20220512141820-create-Achievements-table.ts

```diff
@@ -99,8 +100,8 @@ module.exports = {
 
   async down({ context: { queryInterface } }: SequelizeContext) {
     return queryInterface.sequelize.transaction(async (transaction) => {
-      await queryInterface.dropTable('Achievement', { transaction });
       await queryInterface.dropTable('AchievementUnlocked', { transaction });
+      await queryInterface.dropTable('Achievement', { transaction });
```

Comment:
this is a fix that I found... AchievementUnlocked has to be dropped before Achievement